platform-validation/scripts/compatibility.sh

```diff
@@ -490,14 +531,31 @@ check_systemd(){
     fi
 }
 
+cat /etc/os-release
 get_architecture
 echo "Architecture:$ARCH"
 echo "OS Type:$OSTYPE"
 
-#TODO : Do we need to check in both host and container?
-check_net_cap_bind_host
-check_net_cap_bind_container
 check_cgroup_heirachy
+#Flags Required for setting elevated capabilities in a container.
+check_kernel_flags EXT4_FS_SECURITY
+
+#Todo : Only check if docker engine is not present?
+#Check for Required Container Engine Flags if docker is not present
+check_kernel_flags \
+	NAMESPACES NET_NS PID_NS IPC_NS UTS_NS \
```

Comment:
Can you have a comment for each flag and what the flag corresponds to

Reply:
I added a general comment on why these checks are needed and provided a link to description on the config flags. I don't think we are at a stage where we know exactly how each configuration flag prevents us from running iot edge. We just have an idea that these flags are required for running a container engine which is a requirement for IoT Edge and hence I am hesitant to include any description in the code itself

Reply:
where is the comment?